supertokens_python/querier.py

```diff
@@ -159,17 +200,33 @@ async def __get_headers_with_api_version(self, path: NormalisedURLPath):
         return headers
 
     async def send_get_request(
-        self, path: NormalisedURLPath, params: Union[Dict[str, Any], None] = None
+        self,
+        path: NormalisedURLPath,
+        params: Union[Dict[str, Any], None] = None,
+        user_context: Union[Dict[str, Any], None] = None,
     ) -> Dict[str, Any]:
         if params is None:
             params = {}
 
         async def f(url: str, method: str) -> Response:
+            headers = await self.__get_headers_with_api_version(path)
+            nonlocal params
+            if Querier.network_interceptor is not None:
+                (
+                    url,
+                    method,
+                    headers,
+                    params,
+                    _,
+                ) = Querier.network_interceptor(  # pylint:disable=not-callable
```

Comment:
why `# pylint:disable=not-callable`?

Reply:
Known issue with `pylint`

https://github.com/pylint-dev/pylint/issues/1493#issuecomment-484490930